Project.toml

```diff
@@ -19,12 +20,14 @@ Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 [compat]
 AssetRegistry = "0.1"
 Blink = "0.12"
+FFTW = "1.0"
 Interact = "0.10"
+KomaMRIBase = "0.8"
 KomaMRICore = "0.7"
 KomaMRIFiles = "0.7"
 KomaMRIPlots = "0.7"
 MAT = "0.10"
-MRIReco = "0.6, 0.7"
+MRIReco = "0.8"
```

Comment:
Do we really need to remove compat for MRIReco 0.6, 0.7 ?

Reply:
When I restricted to MRIBase 0.4 and then set to MRIReco = "0.6, 0.7", it throws me an unsatisfiable requirements error:
![image](https://github.com/cncastillo/KomaMRI.jl/assets/36004160/bea05d6a-a104-4839-ad96-28075af38b3e)

Reply:
Ok let's leave just then 0.8